src/jdk.jlink/share/classes/jdk/tools/jmod/resources/jmod.properties

```diff
@@ -82,6 +82,10 @@ main.opt.date=Date and time for the timestamps of entries, specified in ISO-8601
 
 main.opt.cmdfile=Read options from the specified file
 
+main.opt.compress=Compression to use when creating the JMOD archive.\
+\ Accepted values are: zip-[0-9], where zip-0 provides no compression, and zip-9\
+\ provides the best compression.
```

Comment:
Should specify the default if not set.

Reply:
Here is where we get into a minor pickle. We currently default to `Deflater.DEFAULT_COMPRESSION`, which is `-1`, and which is handled inside `zlib` as `6`. We can say `zip-6` here is the default, but that would be stale if zlib ever updates its default. Still fair to say `zip-6` here, or?

Reply:
Actually, nevermind. Let's just put the defacto `6` into `jmod` defaults, and use it. See new commit.

Reply:
Sounds good as the implementation sets 6 as the default.  The new commit looks good.